########## Exception/Error handling

# try 
# except 
# finally

# try:
#     a = 2
#     print(b)
# except:
#     print("Please check")

####################

# try:
#     a = 2
#     b = 0
#     c = a/b

#     print(c)
# except Exception as e:
#     print("Errorrrrr")
#     print(e)

####################
# try:
#     n = int(input("Enter integer: "))
#     print(n)
# except:
#     print("Please enter valid input.")

###################

# try:
#     n = int(input("Enter integer: "))
#     print(n)
# except:
#     print("Please enter valid input.")
# finally:
#     print("Completed")